applications/GeoMechanicsApplication/tests/cpp_tests/test_interface_stress_state_policy.cpp

```diff
@@ -0,0 +1,114 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_elements/interface_stress_state.h"
+#include "custom_geometries/line_interface_geometry.h"
+#include "geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(InterfaceStressState_CloneCreatesCorrectInstance, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    const std::unique_ptr<StressStatePolicy> p_stress_state_policy =
+        std::make_unique<InterfaceStressState>();
+
+    KRATOS_EXPECT_NE(dynamic_cast<InterfaceStressState*>(p_stress_state_policy->Clone().get()), nullptr);
```

Comment:
I would suggest to also check whether the clone is not the same object as the original:
```suggestion
    const auto p_cloned_policy = p_stress_state_policy->Clone();
    KRATOS_EXPECT_NE(dynamic_cast<InterfaceStressState*>(p_cloned_policy.get()), nullptr);
    KRATOS_EXPECT_NE(p_cloned_policy.get(), p_stress_state_policy.get());
```